core/frontend/src/types/ping.ts

```diff
@@ -0,0 +1,19 @@
+import { Dictionary } from '@/types/common'
+
+export enum PingType {
+  Ping1D = 'Ping1D',
+  Ping360 = 'Ping360'
+}
+
+export interface PingDevice {
+  ping_type: PingType,
+  device_id: number,
+  device_model: number,
+  device_revision: number,
+  firmware_version_major: number,
+  firmware_version_minor: number,
+  firmware_version_patch: number,
+  port: string,
+  driver_status: Dictionary<number|boolean|string>,
```

Comment:
As changed in the backend, it would be good to also have a DriverStatus interface definition in the frontend.